src/vcpkg/postbuildlint.cpp

```diff
@@ -1468,7 +1468,7 @@ namespace vcpkg
             error_count += check_no_absolute_paths_in(
                 fs,
                 package_dir,
-                std::vector<Path>{package_dir, paths.installed().root(), paths.build_dir(spec), paths.downloads},
+                std::array<Path, 4>{package_dir, paths.installed().root(), paths.build_dir(spec), paths.downloads},
                 msg_sink);
```

Comment:
How about:

    Path tests[] = {package_dir, paths.installed().root(), paths.build_dir(spec), paths.downloads};
    error_count += check_no_absolute_paths_in(
        fs,
        package_dir,
        tests,
        msg_sink);

to avoid needing to say `4`?